README.md

```diff
@@ -33,7 +33,7 @@ Compared with other graph database solutions, **Nebula Graph** has the following
 
 The Nebula Graph team is glad to announce the Nebula Graph 2.0.0 GA. This is a brand new start with several important updates, and more of them are on the way. Welcome to join us.
 
-This repo hosts the code of versions prior to v1.2.1 and v2.5.0 later. If you are looking to use the versions between v2.0.0 and v2.5.0, please head to [Nebula Graph repo](https://github.com/vesoft-inc/nebula-graph). 
+This repo hosts the code of versions prior to v1.x and v2.5.0 later. If you are looking to use the versions between v2.0.0 and v2.5.0, please head to [Nebula Graph repo](https://github.com/vesoft-inc/nebula-graph). 
```

Comment:
"prior to" indicates before. Readers might mistake it for the 0.x versions. Use "prior to v2.0.0 alpha" or "This repository hosts the source code of Nebula Graph versions before 2.0.0-alpha and after v2.5.0."

Reply:
Thanks, accepted and modified it.